models/model-a/src/code/sg_model.py

```diff
@@ -0,0 +1,30 @@
+from sagemaker.pytorch import PyTorch
+import yaml
+
+# Load YAML configuration
+with open('model-training-config.yaml', 'r') as file:
+    config = yaml.safe_load(file)
+
+# Extract training parameters
+training_params = config["Training"]["Parameters"]
+resources = config["Training"]["Resources"]
+output = config["Training"]["Output"]
+
+# Dynamically set PyTorch Estimator parameters from YAML
+pytorch_estimator = PyTorch(
+    entry_point="train.py",  # Training script
+    source_dir="./code",  # Directory containing train.py and dependencies (e.g., model.py)
+    instance_type=resources["instance_type"],
+    instance_count=resources["instance_count"],
+    framework_version="2.3.0",
+    py_version="py3",
+    script_mode=True,
+    hyperparameters={
+        "epochs": training_params["num_epochs"],
+        "batch-size": training_params["batch_size"],
+        "learning-rate": training_params["learning_rate"],
+    },
+)
+
+pytorch_estimator.fit({'train': 's3://textclassificationmldemo-model-archiving-1734622512866/training/data',
```

Comment:
Change this to:
`s3://textclassificationmldemo-model-archiving-1734622512866/model-a/training/data`
`s3://textclassificationmldemo-model-archiving-1734622512866/model-a/training/test`
Please